public/catalog/kitchensink.yaml

```diff
@@ -108,6 +110,81 @@ nodes:
   label: Node 3
   description: Description 3
   category: grouptests
+- title: Node to test pruning of defaultstest
+  id: defaultprune
+  label: Default prune?
+  category: defaultstest
+  uiSchema:
+    prop4:
+      items:
+        ui:widget: radio
+  schema:
+    type: object
+    additionalProperties: false
+    properties:
+      prop1:
+        title: String with no default
+        type: string
+      prop2:
+        title: String with default empty string
+        type: string
+        default: ''
+      prop3:
+        title: String with default x string
+        type: string
+        default: 'x'
+      prop4:
+        title: Array of booleans
+        type: array
+        items:
+          type: boolean
+          default: false
```

Comment:
Adding an array item and then removing it again, the empty array remains in toml, whereas it wasn't there initially. Is this intended behaviour?

Reply:
duplicate of older comment